Sources_v3/StreamChat/Query/Sorting/ChannelListSortingKey.swift

```diff
@@ -47,3 +47,15 @@ extension ChannelListSortingKey {
         .init(key: rawValue, ascending: isAscending)
     }
 }
+
+/// In case elements have same or `nil` target sorting values it's not possible to guarantee that order of elements will be the same
+/// all the time. So we need to additionally provide safe sorting option.
+extension Array where Element == Sorting<ChannelListSortingKey> {
+    func appendingSafeSortingIfNeeded() -> [Sorting<ChannelListSortingKey>] {
```

Comment:
Why not call it with a more descriptive name, for example `appendingCidSortingKey`? There's nothing like "safe sorting" :) Basically, what we're doing here is to make sure the sorting is always stable by appending a sort descriptor that's guaranteed to be unique.